app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -13,25 +13,25 @@ class UspsLocationsController < ApplicationController
 
       # retrieve the list of nearby IPP Post Office locations with a POST request
       def index
-        usps_response = []
+        response = []
+
         begin
           if IdentityConfig.store.arcgis_search_enabled
             candidate = UspsInPersonProofing::Applicant.new(
               address: search_params['street_address'],
               city: search_params['city'], state: search_params['state'],
               zip_code: search_params['zip_code']
             )
-            usps_response = proofer.request_facilities(candidate)
+            response = proofer.request_facilities(candidate)
           else
-            usps_response = proofer.request_pilot_facilities
+            response = proofer.request_pilot_facilities
           end
-        rescue ActionController::ParameterMissing
-          usps_response = proofer.request_pilot_facilities
-        rescue Faraday::ConnectionFailed => _error
-          nil
+        rescue => err
+          Rails.logger.warn(err)
```

Comment:
I think it would be nice to log this in a more structured way, such as

```
Rails.logger.warn(exception_class: err.class, exception_message: err.message)
```

That way we can easily query these exception messages in cloudwatch later, eg make a table of exception classes by count. I don't think the default stringification of exceptions will make it easy on us if we're trying to sort through hundreds of error messages down the road

And then update the spec to check for a more specific error message instead of just that `warn` was called

Reply:
if we did that, I think we need to throw a `{}.to_json` around that, but otherwise yes, big fan of structured logs